pkg/maps/bwmap/bwmap.go

```diff
@@ -43,8 +41,7 @@ type EdtInfo struct {
 	Pad             [4]uint64 `align:"pad"`
 }
 
-func (v *EdtInfo) GetValuePtr() unsafe.Pointer { return unsafe.Pointer(v) }
-func (v *EdtInfo) String() string              { return fmt.Sprintf("%d", int(v.Bps)) }
+func (v *EdtInfo) String() string { return fmt.Sprintf("%d", int(v.Bps)) }
```

Comment:
I'm trying to avoid altering behaviours like these in this PR. Agree overall, this was likely copypasta'd, but wouldn't cause anything worse than a cosmetical bug.